src/component/SearchResultContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import { flickrFeedItemInterface } from "./utility/extractFlickrFeedData";
+import { SearchResultItem } from "./SearchResultItem";
+
+interface SearchResultContainerProps {
+  data: flickrFeedItemInterface[];
+}
+
+export const SearchResultContainer = ({ data }: SearchResultContainerProps) => {
+  return (
+    <div id="container" className="max-w-screen-xl mx-auto p-x4 py-10">
+      <div className="grid grid-cols-4 gap-6">
+        {data.map((item, idx: number) => {
```

Comment:
Using index here for key as the list will not be removed, deleted or rearranged.